### DISCUSSION TITLE
ReactDOM.render is no longer supported in React 18 > 17

### DISCUSSION THREAD

**Discussion by yosmarcode** [grid-js/gridjs#1363] [General]
I have an error when I try to use the library with the version of "react": "^18.2.0",
next error
createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn

I think the package.json file should be updated to the most recent version of react

I comment it is not an impediment to use it but the error is very annoying

I don't know if I'm doing something wrong

thank you very much first of all

**Comment by ryandowd** [comment]
Has there been any update or fix for this?

**Reply by thallysondias** [reply]
up!

**Reply by yosmarcode** [reply]
The truth is I don't know if they solved it but the issue is in their app directory. library is created based on reactjs 17 But since react from version 18 eliminated several things that are deprecated, it should have been fixed, but it is not an impediment to using the library with the new versions of react
